examples/sink-get-200.json

```diff
@@ -0,0 +1,177 @@
+{
+  "id": "171d5c87-7fff-4c53-9383-46503eb1c23e",
+  "version": "1441704616:592733242",
+  "label": "Display 1",
+  "description": "Display placed on the table in the kitchen",
+  "tags": [],
+  "edid": {
+    "manufacturer": "SAM Model b43",
+    "manufacture_week": 19,
+    "manufacture_year": 2018,
+    "version": "1441704616:592733242",
+    "screen_size": {
+      "width": 53,
+      "height": 30
+    },
+    "gamma": 2.2,
+    "display_type": [ "RGB444", "YUV444" ],
```

Comment:
does not match schema

Reply:
I fixed it in 3d9f28eb2630ae930e502ce6be6eccc863cc54dd.